Tests/AutoCollection/HttpRequestParser.tests.ts

```diff
@@ -122,6 +123,7 @@ describe("AutoCollection/HttpRequestParser", () => {
             var newTags = helper.getRequestTags(originalTags);
             assert.equal(newTags[(<any>HttpRequestParser).keys.locationIp], 'originalIp');
             assert.equal(newTags[(<any>HttpRequestParser).keys.userId], 'originalUserId');
+            assert.equal(newTags[(<any>HttpRequestParser).keys.userAuthUserId], 'originalAuthUserId');
```

Comment:
You might want to add another assertion like this to the next section "should read tags from headers"

Reply:
Yep! I've didn't notice the next section.
Done!

Reply:
@OsvaldoRosado I still dont know why the tests failed...

Reply:
Looks like something changed in the test environment. I'll look into this